lib/rubygems/specification.rb

```diff
@@ -14,6 +14,7 @@
 require 'rubygems/stub_specification'
 require 'rubygems/util/stringio'
 require 'rubygems/util/list'
+require 'rubygems/util/spdx'
```

Comment:
Can you move this inside the `validate` method?  We should avoid the extra memory overhead for something that we only use at `gem build` time.